### PR TITLE
Nav aside home page

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone this is my first pull request for my social media 
Files needed to be reviewed are
src/pages
Home page 
Landing Page
src/componenets/
LeftAside
RightAside
Home Page
captures_chrome-capture-2022-3-19
Landing Page
chrome-capture-2022-3-19

### CODE REVIEW

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { LinkedIn } from "Assets/Icons/LinkedIn";
+import { Twitter } from "Assets/Icons/Twitter";
+function Footer() {
+  return (
+    <footer className="text-gray-600 body-font">
+      <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
+        <a className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+       
+          <img
+            src={require("./../../Assets/Images/FooterImage.png")}
```
Comment (mikesch803) [inline src/Components/Footer/Footer.jsx:12]: this is new to me that we can also import like that, great!

File: src/Components/LeftAside/LeftAside.jsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import styles from "./LeftAside.module.css";
+import { Link, NavLink } from "react-router-dom";
+import { GiHomeGarage } from "react-icons/gi";
+import { HiHashtag } from "react-icons/hi";
+import { BsBell, BsEnvelope } from "react-icons/bs";
+
+import { FaEllipsisH } from "react-icons/fa";
+
+import { HiOutlineDotsCircleHorizontal } from "react-icons/hi";
+import { CgProfile } from "react-icons/cg";
```
Comment (mikesch803) [inline src/Components/LeftAside/LeftAside.jsx:11]: add index file for multiple imports

File: src/Components/LeftAside/LeftAside.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                    <img
+                      src={require("./../../Assets/Images/FooterImage.png")}
+                      alt="logo"
+                    />
+                  </Link>
+                </h1>
+              </div>
+              {/* mt-0.5 */}
+              <div className={`${styles.navigation} mt-2 mb-1 `}>
+                <nav>
```
Comment (mikesch803) [inline src/Components/LeftAside/LeftAside.jsx:31]: suggestion : here navlinks are repeating which you can reduce here

File: src/Pages/HomePage/HomePage.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      <div className="layout__main">
+        <header className=" layout__main-header">
+          <span className="text">Home</span>
+        </header>
+        <div className="empty"></div>
+        <PostInput/>
+        <Post />
+        <Post />
+        <Post />
+        <Post />
```
Comment (mikesch803) [inline src/Pages/HomePage/HomePage.jsx:22]: why post component repeating multiple times?

**Review by mikesch803 — APPROVED** [review]

File: src/Components/Footer/Footer.jsx (hunk 1 above)
Comment (Dev-AkshatGupta) [reply src/Components/Footer/Footer.jsx:12]: Yeah we can do that mahendra

File: src/Components/LeftAside/LeftAside.jsx (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/Components/LeftAside/LeftAside.jsx:11]: Yeah sure will do that

File: src/Pages/HomePage/HomePage.jsx (hunk 4 above)
Comment (Dev-AkshatGupta) [reply src/Pages/HomePage/HomePage.jsx:22]: That is static as of now so will make that functional now

**Review by Hamza-777 — APPROVED** [review]
Looks good.

**Pull request merged**